types/relaxed-json/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "../tslint.json" }
```

Comment:
The generator script is out of date (Microsoft/dts-gen#50), and you may be running an older version of dtslint if you're getting the above error.